### PR TITLE
feat: initial commit

### PR DESCRIPTION
**Pull request by tomhobson**
Added the initial trivydb downloader

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:21.04 as builder
```
Comment (ankitm123) [inline Dockerfile:1]: Can we use alpine or other lightweight images? Also 21.04 is not LTS, so let's not use it, best to stick to LTS versions imo.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+WORKDIR /source
+COPY ./downloader.sh .
+RUN apt update && apt install -y wget curl && bash downloader.sh
```
Comment (ankitm123) [inline Dockerfile:4]: Does update work without `-y`?

Comment (tomhobson) [reply Dockerfile:4]: Yep tested before I pushed :)

Comment (rawlingsj) [reply Dockerfile:4]: @ankitm123 lets get things working as is ATM as its just initial development, and improve with subsequent PRs once pipelines are setup

**Pull request merged**